packages/adapter-utils/src/utils.ts

```diff
@@ -80,7 +80,9 @@ const fieldMapperGenerator = (
     return name => objType.fields[name]
   }
   const objType = new ObjectType({ elemID: new ElemID('') })
-  return name => (type[name] !== undefined ? new Field(objType, name, type[name]) : undefined)
+  return name => (type[name] !== undefined
+    // we set the annotations as type[name].annotations to support hidden_string (or any other type with hidden_value) in instance annotation types.
+    ? new Field(objType, name, type[name], type[name].annotations) : undefined)
```

Comment:
style nit
```suggestion
  return name => (
    type[name] !== undefined
      ? new Field(objType, name, type[name], type[name].annotations)
      : undefined
  )
```

also, this new behavior should be documented with a test - we should add a test that ensures annotation types annotations are copied to field annotations when using `transformElement`